resources/graphql/authorization.graphql

```diff
@@ -56,11 +56,19 @@ input UpdateUserInput {
 type Query {
     "Limit to current user only"
     getMyUser : User
+    listOrganizations: [OrgInfo]
```

Comment:
The return type of listOrganizations should be "Organization". "OrgInfo" is only used as a sub-type for "User"

Reply:
Discussed and determined that we should use OrgInfo